HACKING.md

```diff
@@ -92,6 +92,129 @@ Package build dependencies for other distributions can be found under the
 
 Go module dependencies are automatically resolved at build time.
 
+### Building the snap with snapcraft
+
+The easiest (though not the most efficient) way to test changes to snapd is to
+build the snapd snap using _snapcraft_ and then install that snapd snap. The
+snapcraft.yaml for the snapd snap is located at ./build-aux/snapcraft.yaml, and
+can be built using snapcraft either in a LXD container or a multipass VM (or
+natively with `--destructive-mode` on a Ubuntu 16.04 host).
+
+Note: Currently, snapcraft's default track of 5.x does not support building the 
+snapd snap, since the snapd snap uses `build-base: core`, which uses Ubuntu 
+16.04 as the base for building and Ubuntu 16.04 is in Extended Security 
+Maintenance (ESM), and as such only is buildable using snapcraft's 4.x channel.
```

Comment:
This is a bit hard to read (at least for someone who's not a native speaker), and I'm not even sure that I understand the explanation; maybe we can just remove the details, and say that 5.x does not work, while 4.x does? :-)

Reply:
re-worded to be more clear in https://github.com/snapcore/snapd/pull/11124